src/LibPQ.jl

```diff
@@ -878,6 +878,11 @@ function string_parameters end
 
 string_parameters(parameters::AbstractVector{<:Parameter}) = parameters
 
+# Tuples of parameters
+string_parameters(parameters::Tuple{Vararg{Parameter}}) = collect(parameters)
```

Comment:
I don't believe you need this, since the method above is a no-op on arrays of `Parameter`.

Reply:
This method simply converts the tuple to an array, which as you said is a no-op. Since I assume downstream methods still expect an array of either `String` or `Parameter`, my understanding is that the conversion to an array would have to happen somewhere.

Reply:
Yep, the method you have below will do that.